src/einsteinpy/symbolic/predefined/vacuum_solutions.py

```diff
@@ -115,3 +115,44 @@ def KerrNewman(
     ).tolist()
     metric[0][3] = metric[3][0] = sch * r * a * (sin(theta) ** 2) / (Sigma * c)
     return MetricTensor(metric, coords, "ll", name="KerrNewmanMetric")
+
+
+def Reissner_Nordstorm(
```

Comment:
```suggestion
def ReissnerNordstorm(
```
drop the underscore, as followed by other predefined metrics